components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/consent/ConsentMgtPostAuthnHandler.java

```diff
@@ -153,151 +121,111 @@ protected PostAuthnHandlerFlowStatus handlePreConsent(HttpServletRequest request
                                                           AuthenticationContext context)
             throws PostAuthenticationFailedException {
 
-        String serviceProvider = context.getSequenceConfig().getApplicationConfig().getApplicationName();
+        String spName = context.getSequenceConfig().getApplicationConfig().getApplicationName();
 
         // Due to: https://github.com/wso2/product-is/issues/2317.
         // Should be removed once the issue is fixed
-        if ("DEFAULT".equalsIgnoreCase(serviceProvider)) {
```

Comment:
Shall we add the constant again and use here ?

Reply:
Fixed with 3901421